orm/internal/testkv/leveldb.go

```diff
@@ -3,10 +3,10 @@ package testkv
 import (
 	"testing"
 
-	dbm "github.com/cosmos/cosmos-db"
 	"gotest.tools/v3/assert"
 
 	"cosmossdk.io/orm/model/ormtable"
+	dbm "cosmossdk.io/store/db"
```

Comment:
I think it's fine if orm imports cosmos-db. Nothing here imports orm anyway, and it would allow us to remove the extra file in store